currency-converter-connector-product/zip.xml

```diff
@@ -13,6 +13,7 @@
                 <include>*.json</include>
                 <include>openapi.*</include>
                 <include>README.md</include>
+                <include>README_DE.md</include>
```

Comment:
why not use a wildcard for this? There are maybe more languages that will follow. Why not include them automatically?
```suggestion
                <include>README*.md</include>
```

Reply:
Hi @ivy-lli, thanks for the suggestion, sound good to me.